app/views/phones/_phone_fields.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="nested-fields">
+  <fieldset>
+    <%= phone.input :id, :as => :hidden %>
+    <%= phone.input :category, :as => :select, :collection => Channel::CATEGORIES %>
+    <%= phone.input :content, :placeholder => '9785551212' %>
```

Comment:
Could you please change this to "number" ? See issue #482 
Also, the placeholder text should be +19995551212 ? See issue #483 
Thanks !

Reply:
If you could revert the Rails upgrade, and the two other small changes, that would be great. Thanks for your help!